lib/configure.js

```diff
@@ -461,7 +461,7 @@ PythonFinder.prototype = {
         this.log.silly('stripping "rc" identifier from version')
         version = version.replace(/rc(.*)$/ig, '')
       }
-      var range = semver.Range('>=2.5.0 <3.0.0')
+      var range = semver.Range('>=2.6.0 <3.0.0')
```

Comment:
Should this be 2.7.0 instead of 2.6.0 at this point?  [Python 2.6 has been EOL for 5 years](https://docs.python.org/devguide/index.html#branchstatus).